include/bbp/sonata/config.h

```diff
@@ -174,11 +181,14 @@ class SONATA_API CircuitConfig
     // manifest variables
     std::string _expandedJSON;
 
+    // Config Type
+    Type _circuitType = Type::Complete;
```

Comment:
I feel a better name than `Type` should exist. I'll update the comment if I find something.

Reply:
I also dislike it; I had `CircuitConfigType`, but that seemed redundant.  `Filled` / `Completeness`? I'm also not coming up with anything satisfactory.

Reply:
OTOH, it looks like what the setting really affects, is if the file is checked to be conforming/complete. So maybe the setting could be related to checking: `CheckCompleteness`, `Validate`, `QuickFail`, `AssertMode`, `LazyChecking`, `AllowMissing`.

Reply:
`Status`? Seems not really a `type` qualifier, but really something about the _state_ of the circuit.

Reply:
I went with `ConfigStatus`; not the happiest with that, but hopefully it conveys the meaning.